R/module_nested_tabs.R

```diff
@@ -291,10 +293,14 @@ srv_nested_tabs.teal_module <- function(id, datasets, modules, is_module_specifi
     get_datasets_code(datanames, datasets, hashes)
   )
 
-  do.call(
+
+  data <- do.call(
     teal.data::teal_data,
     args = c(data, code = list(code), join_keys = list(datasets$get_join_keys()[datanames]))
   )
+
+  data@verified <- attr(datasets, "verification_status")
+  return(data)
```

Comment:
Per NEST convention we don't use explicit returns at the end of functions.

Reply:
I'll squeeze this in the pre-release activities PR which looks for checks and conventions like these. Thanks!

Reply:
Fixed here https://github.com/insightsengineering/teal/pull/1021/commits/d11315ac1aec9d8703ec88b5c8ba447d926028c4